lib/tracking_options.dart

```diff
@@ -67,38 +65,27 @@ class TrackingOptions {
     this.idfv = true
   });
 
-  Map<String, Set<String>> toMap() {
-    Set<String> disableFields = {};
-
-    // General tracking options
-    if (!ipAddress) disableFields.add(Constants.ampTrackingOptionIpAddress);
-    if (!language) disableFields.add(Constants.ampTrackingOptionLanguage);
-    if (!platform) disableFields.add(Constants.ampTrackingOptionPlatform);
-
-    // Mobile tracking options
-    if (!region) disableFields.add(Constants.ampTrackingOptionRegion);
-    if (!dma) disableFields.add(Constants.ampTrackingOptionDma);
-    if (!country) disableFields.add(Constants.ampTrackingOptionCountry);
-    if (!city) disableFields.add(Constants.ampTrackingOptionCity);
-    if (!carrier) disableFields.add(Constants.ampTrackingOptionCarrier);
-    if (!deviceModel) disableFields.add(Constants.ampTrackingOptionDeviceModel);
-    if (!deviceManufacturer) disableFields.add(Constants.ampTrackingOptionDeviceManufacturer);
-    if (!osVersion) disableFields.add(Constants.ampTrackingOptionOsVersion);
-    if (!osName) disableFields.add(Constants.ampTrackingOptionOsName);
-    if (!versionName) disableFields.add(Constants.ampTrackingOptionVersionName);
-
-    // Android-specific tracking options
-    if (!adid) disableFields.add(Constants.ampTrackingOptionAdid);
-    if (!appSetId) disableFields.add(Constants.ampTrackingOptionAppSetId);
-    if (!deviceBrand) disableFields.add(Constants.ampTrackingOptionDeviceBrand);
-    if (!latLag) disableFields.add(Constants.ampTrackingOptionLatLag);
-    if (!apiLevel) disableFields.add(Constants.ampTrackingOptionApiLevel);
-
-    // iOS-specific tracking options
-    if (!idfv) disableFields.add(Constants.ampTrackingOptionIdfv);
-
+  Map<String, bool> toMap() {
     return {
-      'disabledFields': disableFields,
```

Comment:
How we disable the tracking options now?

Reply:
It follows browser SDK pattern. See the tests at 'test/tracking_options_test.dart' and more context around the change [here](https://github.com/amplitude/Amplitude-Flutter/pull/162#discussion_r1478923413).

Reply:
Thanks for the content.